src/main/java/org/gridsuite/shortcircuit/server/dto/ShortCircuitParametersInfos.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.shortcircuit.server.dto;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonProperty.Access;
+import com.powsybl.shortcircuit.ShortCircuitParameters;
+import com.powsybl.shortcircuit.VoltageRange;
+import lombok.Builder;
+import lombok.extern.jackson.Jacksonized;
+import org.gridsuite.shortcircuit.server.service.ShortCircuitService;
+
+import java.util.List;
+
+/**
+ * @author AJELLAL Ali <ali.ajellal@rte-france.com>
```

Comment:
you ?

Reply:
Nope, that's not me who created this class, I just move the file.